src/components/link-component/LinkComponent.tsx

```diff
@@ -0,0 +1,39 @@
+import { FC } from 'react'
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+import LinkOutlinedIcon from '@mui/icons-material/LinkOutlined'
+
+import AppButton from '~/components/app-button/AppButton'
+
+import { ButtonVariantEnum, Link } from '~/types'
+import { styles } from '~/components/link-component/LinkComponent.styles'
+
+interface LinkComponentProps {
+  link: Link
+}
+
+const openLink = (link: Link) => {
```

Comment:
you can also move this function inside and don`t pass link as a parameter. Name this function as onClick

Reply:
This openLink appears in two components

Reply:
> This openLink appears in two components

moved to constants